xbmc/interfaces/python/AddonPythonInvoker.h

```diff
@@ -16,8 +16,9 @@ class CAddonPythonInvoker : public CPythonInvoker
   explicit CAddonPythonInvoker(ILanguageInvocationHandler *invocationHandler);
   ~CAddonPythonInvoker() override;
 
+  static void globalInitializeModules(void);
```

Comment:
```suggestion
  static void GlobalInitializeModules();
```

ref: https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#86-methods
and: https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#36-superfluous-void

Reply:
@lrusak, this is updated in the last commit.